build.gradle

```diff
@@ -19,6 +19,7 @@ repositories {
     mavenLocal()
     mavenCentral()
     maven { url 'https://jitpack.io' }
+    maven { url 'https://pkgs.dev.azure.com/hmcts/Artifacts/_packaging/hmcts-lib/maven/v1' }
```

Comment:
Can jitpack.io be removed?

Reply:
I keep https://jitpack.io/ in finrem-ccd-definitions temporary as it causes:

Could not resolve com.github.hmcts:befta-fw:9.2.3
https://build.hmcts.net/blue/organizations/jenkins/HMCTS_d_to_i%2Ffinrem-ccd-definitions/detail/PR-2534/7/pipeline